compiler/cgen.nim

```diff
@@ -1290,82 +1314,115 @@ proc genInitCode(m: BModule) =
     # Keep a bogus frame in case the code needs one
     add(prc, ~"\tTFrame FR_; FR_.len = 0;$N")
 
+    if m.preInitProc.s(cpsLocals).len > 0:
```

Comment:
git diff algorithm failure here: there is no new lines here, it is existing lines wrapped into `if rope.len > 0: initRequired = true`